packages/lib/WebDavApi.ts

```
@@ -410,7 +449,8 @@ class WebDavApi {
 		// The "solution", an ugly one, is to send a purposely invalid string as eTag, which will bypass the If-None-Match check  - Seafile
 		// finds out that no resource has this ID and simply sends the requested data.
 		// Also add a random value to make sure the eTag is unique for each call.
-		if (['GET', 'HEAD'].indexOf(method) < 0) headers['If-None-Match'] = `JoplinIgnore-${Math.floor(Math.random() * 100000)}`;
+		if (['GET', 'HEAD'].indexOf(method) < 0 && this.excludeIfNoneMatch !== ExcludeIfNoneMatch.Yes) 
+			{ headers['If-None-Match'] = `JoplinIgnore-${Math.floor(Math.random() * 100000)}`; }
```

ESLint

[error] Expected indentation of 2 tabs but found 3.

(`@typescript-eslint/indent`)

---

[error] Expected indentation of 2 tabs but found 3.

(indent)